### PR TITLE
Added author to RSS, made more configurable, throw error in createAssets for missing dir

### PR DESCRIPTION
**Pull request by morganrallen**
Added author tag to RSS feed
Created config.js for server to read config (port, builddir) from.
createAssets didn't throw error if builddir was missing (instead threw exception on files.sort)

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -116,6 +116,8 @@ function createAssets (configpath, builddir, cb) {
       ;
   
     fs.readdir(path.join(builddir, 'db'), function (err, files) {
+	  if(err) throw err;
```
Comment (mikeal) [inline main.js]: we can't be throwing this error on production, maybe just log it instead?

File: server.js [hunk 2]
```diff
@@ -1 +1,2 @@
-require('./main').run(80, '/home/node/planetbuild');
\ No newline at end of file
+var config = require("./config");
+require('./main').run(config.port, config.buildPath);
```
Comment (mikeal) [inline server.js:2]: server.js needs to default to a post that is 80 because it's what no.de runs, edit debug.js if you want to run with a non-80 port.

**Comment by mikeal** [comment]
If you fix those comments I can merge it. I like the site.author being added to the feed.

**Comment by morganrallen** [comment]
OK, made those changes. I still think createAssets needs to be a little more robust, if there is an error at all, files.sort will still throw an exception.

**Pull request merged**